### PR TITLE
VAS-43: Add Unit Tests for events controller and EventTypeLabelView

### PR DESCRIPTION
**Pull request by vmartos**

### CODE REVIEW

File: VAS.Tests/Services/TestEventsController.cs [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+		[Test]
+		public void TestLoadTimelineEvents ()
+		{
+			IEnumerable<TimelineEvent> eventList = new List<TimelineEvent> { ev1, ev2 };
+
+			App.Current.EventsBroker.Publish (new LoadTimelineEvent<IEnumerable<TimelineEvent>> {
+				Object = eventList
+			});
+
+			playerController.Verify (p => p.LoadPlaylistEvent (It.IsAny<Playlist> (), It.IsAny<IPlaylistElement> (), It.IsAny<bool> ()), Times.Once);
```
Comment (dfernandezramos) [inline VAS.Tests/Services/TestEventsController.cs]: Is it possible to check that the Playlist parameter contains 2 TimelineEvents? Or better... contains evt1 and evt2.

File: VAS.Tests/Services/TestEventsController.cs [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+		}
+
+		[Test]
+		public void TestLoadTimelineEvent ()
+		{
+			App.Current.EventsBroker.Publish (new LoadTimelineEvent<TimelineEvent> {
+				Object = ev1
+			});
+
+			playerController.Verify (p => p.LoadEvent (It.IsAny<TimelineEvent> (), It.IsAny<Time> (), It.IsAny<bool> ()), Times.Once);
```
Comment (dfernandezramos) [inline VAS.Tests/Services/TestEventsController.cs]: You could check that the TimelineEvent parameter is evt1

File: VAS.Tests/Services/TestEventsController.cs [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+				Model = ev2,
+				Visible = true
+			});
+
+			eTypeVM.LoadEventType ();
+
+			playerController.Verify (p => p.LoadPlaylistEvent (It.Is<Playlist> (pl => pl.Elements.Count == 1),
+															   It.Is<IPlaylistElement> (pe => ComparePlaylistElement (pe, ev2)),
+															   It.IsAny<bool> ()), Times.Once);
+
```
Comment (dfernandezramos) [inline VAS.Tests/Services/TestEventsController.cs]: Extra line

File: VAS.Tests/Services/TestEventsController.cs [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+				Model = ev2,
+				Visible = true
+			});
+
+			eTypeVM.LoadEventType ();
+
+			playerController.Verify (p => p.LoadPlaylistEvent (It.Is<Playlist> (pl => pl.Elements.Count == 2),
+															   It.Is<IPlaylistElement> (pe => ComparePlaylistElement (pe, ev1)),
+															   It.IsAny<bool> ()), Times.Once);
+
```
Comment (dfernandezramos) [inline VAS.Tests/Services/TestEventsController.cs]: Extra line

**Comment by dfernandezramos** [comment]
Also, all tests must start with "Test" (For example TestFoo) :)

**Comment by vmartos** [comment]
All tests starts with Test... 
The only method that doesn't is:
bool ComparePlaylistElement (IPlaylistElement element, TimelineEvent ev)
that is a private utility method, not a test

File: VAS.Tests/Drawing/TestEventTypeLabelView.cs [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+			eventTypeLabel.ViewModel.ViewModels [0].Visible = false;
+			eventTypeLabel.ViewModel.ViewModels [1].Visible = false;
+
+			Assert.IsTrue (button.Insensitive);
+
+			//Finally Make Only One Event Visible
+			eventTypeLabel.ViewModel.ViewModels [1].Visible = true;
+
+			Assert.IsFalse (button.Insensitive);
+
```
Comment (dfernandezramos) [inline VAS.Tests/Drawing/TestEventTypeLabelView.cs]: Extra line

**Comment by dfernandezramos** [comment]
Not on VAS.Tests/Drawing/TestEventTypeLabelView.cs, Tests doesn't start with "Test"

**Pull request merged**